src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -106,124 +102,149 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
-          </div>
-          <div id="tooltip">
-            <div class="title">${name}</div>
-            <div class="current-temperature">
-              <span class="current-temperature-text">
-                ${stateObj.attributes.current_temperature}
-                ${stateObj.attributes.current_temperature
-                  ? html`
-                      <span class="uom"
-                        >${this.hass.config.unit_system.temperature}</span
-                      >
-                    `
-                  : ""}
-              </span>
-            </div>
-            <div class="climate-info">
-              <div id="set-temperature">
-                ${!this._setTemp
-                  ? ""
-                  : Array.isArray(this._setTemp)
-                  ? html`
-                      ${this._setTemp[0].toFixed(1)} -
-                      ${this._setTemp[1].toFixed(1)}
-                    `
-                  : html`
-                      ${this._setTemp.toFixed(1)}
-                    `}
-              </div>
-              <div class="current-mode">
-                ${stateObj.attributes.hvac_action
-                  ? this.hass!.localize(
-                      `state_attributes.climate.hvac_action.${
-                        stateObj.attributes.hvac_action
-                      }`
-                    )
-                  : this.hass!.localize(`state.climate.${stateObj.state}`)}
-                ${stateObj.attributes.preset_mode &&
-                stateObj.attributes.preset_mode !== CLIMATE_PRESET_NONE
-                  ? html`
-                      -
-                      ${this.hass!.localize(
-                        `state_attributes.climate.preset_mode.${
-                          stateObj.attributes.preset_mode
-                        }`
-                      ) || stateObj.attributes.preset_mode}
-                    `
-                  : ""}
-              </div>
-              <div class="modes">
-                ${(stateObj.attributes.hvac_modes || [])
-                  .concat()
-                  .sort(compareClimateHvacModes)
-                  .map((modeItem) => this._renderIcon(modeItem, mode))}
+        <paper-icon-button
+          icon="hass:dots-vertical"
+          class="more-info"
+          @click=${this._handleMoreInfo}
+        ></paper-icon-button>
+
+        <div id="controls">
+          <div id="slider">
+            ${slider}
+            <div id="slider-center">
+              <div id="temperature">
+                ${currentTemperature} ${setValues}
               </div>
             </div>
           </div>
         </div>
+        <div id="info">
+          <div id="modes">
+            ${(stateObj.attributes.hvac_modes || [])
+              .concat()
+              .sort(compareClimateHvacModes)
+              .map((modeItem) => this._renderIcon(modeItem, mode))}
+          </div>
+          ${name}
+        </div>
       </ha-card>
     `;
   }
 
+  protected rescale_svg() {
```

Comment:
Why `protected`?

Reply:
Because I have no idea what I'm doing.

Reply:
I know that game 😂

Reply:
`protected` means a class that extends this class can overwrite it, so not needed here :-)

Reply:
So here it should be `private`, private class members can be accessed from within the class only.